src/templates/sermons.factory.js

```diff
@@ -0,0 +1,40 @@
+const path = require(`path`),
+    constants = require(`../constants.js`)
+
+exports.createPages = async (graphql, createPage) => {
+    await Promise.all(Object.keys(constants.languages).map(async (key) => {
+        const lang = constants.languages[key]
+
+        const result = await graphql(`
+{
+    avorg {
+        sermons(language:${key}) {
+            nodes {
+                title
+            }
+            pageInfo {
+                hasNextPage
+                hasPreviousPage
+                startCursor
+                endCursor
+            }
+            aggregate {
+                count
+            }
+        }
+    }
+}
+`)
+        const nodes = result &&
+            result.data &&
+            result.data.avorg &&
+            result.data.avorg.sermons &&
+            result.data.avorg.sermons.nodes
```

Comment:
I don't know if you can use optional chaining (`?.`), but if you can't can you import lodash and use [`_.get()'](https://lodash.com/docs/4.17.15#get)?